### ISSUE TITLE
Intermediate HTML and CSS: The Advanced Selectors article has inaccurate example code

### ISSUE THREAD

**Issue by Tenzenmaster** [TheOdinProject/curriculum#28775]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

### Describe your suggestion

[Advanced Selectors](https://github.com/TheOdinProject/curriculum/blob/928844cb02b766e75806b4abd20f5cad3023793b/intermediate_html_css/intermediate_css_concepts/advanced_selectors.md) has some example code that doesn't line up with its comments.
```
/* This rule will only select the div with the class child group2 */
.group1 + div {
  /* Our cool CSS */
}

/* This rule will only select the div with the class child group3 */
.group1 + div + div {
  /* More cool CSS */
}
```
The comments reference nonexistent classes "group2" and "group3", state '+' will select elements with children of a certain class instead of an adjacent sibling, and the second example does not mention how there are two "div"s chained together. These comments should be updated to correctly reflect the CSS.

### Path

Ruby / Rails, Node / JS

### Lesson Url

https://www.theodinproject.com/lessons/node-path-intermediate-html-and-css-advanced-selectors

### (Optional) Discord Name

Tenzenmaster

**Comment by MaoShizhong** [comment]
```html
<main class="parent">
  <div class="child group1">
    <div class="grand-child group1"></div>
  </div>
  <div class="child group2">
    <div class="grand-child group2"></div>
  </div>
  <div class="child group3">
    <div class="grand-child group3"></div>
  </div>
</main>
```

The comments are accurate:
- `.group1 + div` selects any div that immediately follows an element with the class `group1`. The second div of main has the classes `child` and `group2`, which is what the comment says will be the only div selected.
- `.group1 + div + div` selects any div that immediately follows a div that immediately follows an element with the class `group1`, which would match the last div of main, the one with the classes `child` and `group3`.

That being said, I think the comments could be tweaked for clarity e.g. 
```css
/* Selects only the div with the classes "child group2" */
.group1 + div {
  /* Our cool CSS */
}

/* Selects only the div with the classes "child group3" */
.group1 + div + div {
  /* More cool CSS */
}
```
Does that make the comments clearer for you?

**Comment by MaoShizhong** [comment]
## Acceptance criteria

- [ ] The comments for the relevant CSS code example are amended as per the above.

Please comment if you'd like to be assigned to this issue. Please do not open a PR unless you have been assigned by a maintainer.

**Comment by Strange-can** [comment]
I would like to be assigned.

**Comment by MaoShizhong** [comment]
All yours @Strange-can
